app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -150,12 +159,35 @@ class BrowserToolbarIntegration(
 
         customTabsFeature?.start()
         navigationButtonsIntegration?.start()
+        observerSecurityIndicatorChanges()
+    }
+
+    private fun observerSecurityIndicatorChanges() {
```

Comment:
For now this workaround in Focus may be okay. But we should consider moving this behavior to A-C (into the `Presenter`) to avoid having potentially different behavior implementations in our apps (of course this needs to be configurable, since Fenix doesn't need it for now). In A-C, in the `Presenter`, we can also cover it with tests.. :)